app/notifications/rest.py

```diff
@@ -136,7 +136,11 @@ def send_notification(notification_type):
         reply_to_text=template.reply_to_text,
     )
     if not simulated:
-        queue_name = QueueNames.PRIORITY if template.process_type == PRIORITY else None
+        queue_name = (
+            QueueNames.PRIORITY
+            if template.process_type == TemplateProcessType.PRIORITY
+            else None
+        )
```

Comment:
Note for the future:  seeing this reminds me that we're doing away (if we haven't already) with the notion of a priority queue vs. regular queue - it's like this extra bit of processing (and therefore the `TemplateProcessType.PRIORITY` enum) would just go away or be simplified.

I think we had done some work to remove the priority queue already, we'll need to check.

Reply:
I think there is some logic floating around that still relates to the Priority Queue, and a ticket to remove it.